src/main/java/heronarts/lx/midi/surface/MidiFighterTwister.java

```diff
@@ -1009,15 +1010,19 @@ public void controlChangeReceived(MidiControlChange cc) {
       case CHANNEL_ROTARY_ENCODER:
         if (number >= DEVICE_KNOB && number <= DEVICE_KNOB_MAX) {
           int iKnob = number - DEVICE_KNOB;
+          if (value < KNOB_DECREMENT_VERYFAST || value > KNOB_INCREMENT_VERYFAST) {
```

Comment:
I would leave this whole block in the `else` like it was before. Handle all the valid values first, then have this error block.

Doing the "is-it-an-error" conditional check before the valid values seems a bit more confusing to read through.

Reply:
Sounds good.  Fixed.